blockchain/genesis.go

```diff
@@ -244,6 +244,9 @@ func SetupGenesisBlock(db database.DBManager, genesis *Genesis, networkId uint64
 		if storedcfg.Governance.Reward.ProposerUpdateInterval != 0 {
 			params.SetProposerUpdateInterval(storedcfg.Governance.Reward.ProposerUpdateInterval)
 		}
+		if storedcfg.Governance.Reward.MinimumStake.Cmp(common.Big0) != 0 {
+			params.SetMinimumStakingAmount(storedcfg.Governance.Reward.MinimumStake)
+		}
```

Comment:
I think `0` is supposed to be allowed as a minimum stake. 
Also, we need to check whether `storedcfg.Governance.Reward` is nil or `storedcfg.Governance.Reward.MinimumStake` is nil.

Reply:
I have updated it. Thanks.